test/Driver/response-file.swift

```diff
@@ -24,6 +24,19 @@
 // RUN: %target-build-swift -typecheck @%t.6.resp 2>&1 | %FileCheck %s -check-prefix=RECURSIVE
 // RECURSIVE: warning: result of call to 'abs' is unused
 
+// RUN: python -c 'for i in range(500001): print "-DTEST5_" + str(i)' > %t.7.resp
+// RUN: %target-build-swift -typecheck @%t.7.resp %s 2>&1 | %FileCheck %s -check-prefix=LONG
+// LONG: warning: result of call to 'abs' is unused
+
+// RUN: python -c 'for i in range(500001): print "-DTEST6_" + str(i)' > %t.8.resp
+// RUN: %empty-directory(%t/tmp)
+// RUN: env TMPDIR=%t/tmp/ %target-swiftc_driver %s @%t.8.resp -save-temps
```

Comment:
Please include something for TEST6 to test, so that we know the arguments are actually getting read and there isn't just some kind of silent success.

Reply:
TEST6 is specifically testing that the temporary response files are being saved with `-save-temps`. TEST5 uses an input identical to that of TEST6, and it does test that the flags are being read. That being said, does TEST6 need to test that the flags are being read?

Reply:
Hm, that wasn't obvious to me. What if you grouped this RUN line with the previous ones and reused TEST5's response file?

(Also noticing that the response files and the compilation conditions should be using the same numbering to avoid confusion.)